src/styles/header/cards.scss

```diff
@@ -0,0 +1,104 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: $card-width;
+  background-color: white;
+  align-items: center;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  font-family: $font-family;
+  border: 1px solid #f3f3f3;
+  transition: transform 300ms ease;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    color: $color-title;
+    margin: 0 0 4px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    transition: 0.2s;
+
+    &:hover {
+      color: $hover_text_color;
+      transition: 0.2s;
+    }
+  }
+
+  &__code {
+    margin: 0 0 16px;
+    font-size: 10px;
+    font-weight: 400;
+    align-self: flex-start;
+    line-height: 14px;
+    color: $color-code;
+  }
+
+  &__review-container {
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__review-count {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 1px;
+    justify-content: center;
+  }
+
+  &__price-container {
+    width: 100%;
+    height: 18px;
+    margin: 0 0 16px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    line-height: 18px;
+  }
+
+  &__price-title {
+    font-size: 12px;
+    color: $color-buy;
+  }
+
+  &__price {
+    font-size: 16px;
+    font-weight: 700;
+  }
+
+  &__button {
+    box-sizing: border-box;
+    width: 100%;
+    height: 40px;
+    background-color: $color-buy;
+    border-radius: 5px;
+    font-size: 13px;
+    font-weight: 700;
+    color: white;
+    border: none;
+    text-transform: uppercase;
+    text-align: center;
+    cursor: pointer;
+    transition: 0.5s;
+
+    &:hover {
+      background-color: $white_color;
+      color: $color-buy;
+      cursor: pointer;
+      transition: 0.5s;
```

Comment:
It's better to specify to what properties you need to apply the transition.

![image](https://github.com/mate-academy/layout_catalog/assets/54860641/600f95c9-3075-49a2-828a-24369e804f5e)